db/schema.rb

```diff
@@ -58,8 +60,10 @@
     t.string   "type"
     t.integer  "parent_id"
     t.integer  "min_children_needed"
+    t.datetime "deleted_at"
   end
 
+  add_index "approvals", ["deleted_at"], name: "index_approvals_on_deleted_at", using: :btree
```

Comment:
some of the schema changes in this PR make it seem like this was branched from #734 which might have implications for the order in these PRs get merged? That's a yellow flag for me but mostly just about mechanics not substance.